spec/unit/provider/git_spec.rb

```diff
@@ -267,6 +267,42 @@
     end
   end
 
+  context "with a user id" do
+    let(:deploy_user)  { 123 }
+    let(:expected_cmd) { 'git clone  "git://github.com/opscode/chef.git" "/my/deploy/dir"' }
```

Comment:
there's a stray space between `clone` and `"git://` which won't be helping the specs.